tinyiiod.c

```diff
@@ -77,6 +77,16 @@ int32_t tinyiiod_read_line(struct tinyiiod *iiod, char *buf, size_t len)
 	return iiod->ops->read_line(buf, len);
 }
 
+int32_t tinyiiod_read_nonblocking(struct tinyiiod *iiod, char *buf, size_t len)
```

Comment:
are `tinyiiod_read_nonblocking()` & `tinyiiod_read_wait()` used only in `tinyiiod_do_writebuf()` ?
if so, then maybe we should defer adding them, until we find a better use-case for them

Reply:
They are only used here. tinyiiod_read_nonblocking() prepares the serial interface for a big block of data, since there is no handshake here, like on network interface. tinyiiod_read_wait() waits for entire block of data.